apps/examples/testcase/ta_tc/wifi_manager/utc/utc_wifi_manager_main.c

```diff
@@ -0,0 +1,348 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <pthread.h>
+#include <sys/time.h>
+#include <errno.h>
+#include <apps/shell/tash.h>
+#include <wifi_manager/wifi_manager.h>
+#include "../../../tc_common.h"
+
+extern sem_t tc_sem;
+extern int working_tc;
+
+static void wifi_sta_connected(void);		// in station mode, connected to ap
+static void wifi_sta_disconnected(void);	// in station mode, disconnected from ap
+static void wifi_softap_sta_joined(void);	// in softap mode, a station joined
+static void wifi_softap_sta_left(void);		// in softap mode, a station left
+static void wifi_sta_scan_ap_done(wifi_manager_scan_info_s **scan_info, wifi_manager_scan_result_e res); // in station mode, scanning ap is done
+
+static wifi_manager_cb_s wifi_callbacks = {
+	NULL,	//wifi_sta_connected,
+	NULL,	//wifi_sta_disconnected,
+	NULL,	//wifi_softap_sta_joined,
+	NULL,	//wifi_softap_sta_left,
+	wifi_sta_scan_ap_done,	// in station mode, this callback function is called when scanning ap is done.
+};
+
+static wifi_manager_cb_s wifi_null_callbacks = {
+	NULL,	//wifi_sta_connected,
+	NULL,	//wifi_sta_disconnected,
+	NULL,	//wifi_softap_sta_joined,
+	NULL,	//wifi_softap_sta_left,
+	NULL,	// in station mode, this callback function is called when scanning ap is done.
+};
+
+static void wifi_sta_scan_ap_done(wifi_manager_scan_info_s **scan_info, wifi_manager_scan_result_e res)
+{
+	printf("[pkes] T%d --> %s\n", getpid(), __FUNCTION__);
```

Comment:
please remove pkes which is personal log.

Reply:
I remove it.